lib/helpers/display.js

```diff
@@ -178,34 +178,51 @@ const locationComparator = (a, b) => {
 };
 
 export function printOccurrences(bomJson) {
-  const data = [["Group", "Name", "Version", "Occurrences"]];
   if (!bomJson || !bomJson.components) {
+    console.error("Invalid or empty SBOM data.");
     return;
   }
-  for (const comp of bomJson.components) {
-    if (!comp.evidence || !comp.evidence.occurrences) {
-      continue;
-    }
-    data.push([
-      comp.group || "",
-      comp.name,
-      comp.version || "",
-      comp.evidence.occurrences
-        .map((l) => l.location)
-        .sort(locationComparator)
-        .join("\n"),
-    ]);
-  }
+
+  const data = [["Group", "Name", "Version", "Occurrences"]];
+
   const config = {
-    header: {
-      alignment: "center",
-      content: "Component Evidence\nGenerated with \u2665  by cdxgen",
+    columnDefault: {
+      width: 50,
     },
+    columnCount: 4,
+    columns: [{ width: 20 }, { width: 40 }, { width: 50 }, { width: 25 }],
```

Comment:
Is this tested? Width 25 in column 4 for filenames might be too small.

Reply:
> Is this tested? Width 25 in column 4 for filenames might be too small.

I've removed the redundant error log as suggested and adjusted the width for filenames in column 4 to better accommodate longer names.
Here is how it looks:

<img width="1414" alt="Screenshot 2024-12-05 at 3 57 12 PM" src="https://github.com/user-attachments/assets/f0ed4aea-ce38-4955-9dd1-4df59219ef09">